### ISSUE TITLE
Sidebar upgrade prompt: Make whole thing clickable

### ISSUE THREAD

**Issue by beaulebens** [Automattic/wp-calypso#21047]
When we present a prompt to upgrade in the sidebar (in this case for a free Jetpack site):

![plans_ _jetpack_beta_ _wordpress_com](https://user-images.githubusercontent.com/108942/34276150-5ab27cda-e65d-11e7-841e-adbe967704f3.png)

the entire thing should be clickable, rather than requiring the user to find/click the "GO" on the right.

**Comment by beaulebens** [comment]
Closed in #21230
